rules/TrailingWhitespaceRule.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2013-2014 Will Thames <will@thames.id.au>
```

Comment:
that is present in [ansible-lint](https://github.com/willthames/ansible-lint/blob/master/lib/ansiblelint/rules/TrailingWhitespaceRule.py) anyways, is there a reason to include it here?